community/team_members.md

```diff
@@ -70,4 +57,22 @@ next month.
 Udayan is responsible for:
 
 - applying to RGSoC
-- administering RGSoC
\ No newline at end of file
+- administering RGSoC
+
+
+Former Members
+--------------
+
+__[Maximilian Scholz](https://github.com/sims1253) (Leader):__
+
+"Hi, I'm Max and sometimes called the Secretary of Offtopic (mainly by myself).
+I joined coala in early 2015 but real contributions started 2016. I mainly
+work on coala by giving input on design questions helping with administrative
+tasks. My biggest impact on coala so far probably was the community work during
+the second GSoC coala participated in and my conference visits. From that time
+the title Secretary of Offtopic stems."
+
+Max was responsible for:
+
+- Organizing the team
+- GSoC administering
```

Comment:
so are you resigning from this responsibility?
And it looks like you are resigning from the team, not just the team lead like the commit shortlog/PR title says